conf/core/ConfigOptions.asciidoc

```diff
@@ -3636,7 +3636,7 @@ configuration option, 'language.translation.translator', controls which translat
 === poi.search.radii
 
 * Data Type: string
-* Default Value: ``
+* Default Value: `${HOOT_HOME}/conf/core/PoiSearchRadii.json`
```

Comment:
@bwitham is this an ok change?

Reply:
No, that actually won't work. I think you're code is out of date. `poi.search.radii` is meant hold only JSON and `poi.search.radii.config` holds a file. `poi.search.radii` overrides `poi.search.radii.config`. The idea is that `poi.search.radii.config` has the default inital values used by core and services at the start, and then services can override what's in that file using `poi.search.radii` JSON with the user's distance changes.

https://github.com/ngageoint/hootenanny/blob/5b833739318810aeb73b6f822acdc084d3b0681a/hoot-js/src/main/cpp/hoot/js/conflate/point/PoiSearchRadiusJs.cpp#L61

Reply:
I think the intent here was to set the default values for `poi.search.radii` to the contents of the `PoiSearchRadii.json` file, and present those in the web UI to be overridden by the user.  The UI ends up passing json back to the hoot command, not a file reference.  I'm not sure this update to the config doc is needed.

Reply:
Yea I send json to `poi.search.radii ` but I need to send the defaults to the UI like brian said. I was trying to make the script that builds all the advanced options generic so I made these changes instead of doing something like 
```
if (param === 'poi.search.radii')
    // get defaults for `poi.search.radii.config`
```

Reply:
Still not following why the change to `poi.search.radii` is needed. Maybe b/c I'm not getting what this means:

"The update to config doc is needed because the doc gets parsed to a json file and that file is read in the python script"

This won't work?:

- have services read the contents of the file pointed to by `poi.search.radii.config` upon first request only
- add those contents to the UI as the starting default value for `poi.search.radii` to be passed back to the services
- update `poi.search.radii` by the UI as needed with values changed by the user to be passed back to services

If we want to retain the distnce values per user between user sessions, then that would need a different change. The core JSON file will always be tied directly to values obtained from experimental results in the regression tests and isn't user editable from the UI.

Reply:
So currently there is this script that gets run on services build called `scripts/services/CreateConflationTypeGroups.py` and it does this:
```
conf/services/conflationHoot2Ops.json: scripts/services/CreateConflationTypeGroups.py conf/services/conflationTypes.json conf/core/ConfigOptions.json conf/services/conflationTypeDefaults.json
	python $^ $@ || (rm -f $@ ; exit -1)
```

the file `conflationHoot2Ops.json` is used for creating the advanced options in the UI.
For the defaults i split the json object and make an input for each seperate object and give it a default.


The issue with the approach you mentioned is in the services code we would have to add specific logic to get the defaults, inject them in the already made conflationHoot2Ops.json object. The reason I dont like this approach is because if in the future there is another option that is also expecting json we have to add that logic into the services as well and it turns into a list of 'get default options for this `1 param (poi.search.radii.config) and add them in for this other param(poi.search.radii) because in the ui we need to associate default values to the param that will be used for the command.

Reply:
Ok, I think I know what you're saying now. I forgot there was that intermediate step using the Python to generate the file. I'm just going to get rid of `poi.search.radii.config`. Then `poi.search.radii` can have either the JSON or the file. Should all still work. I'll push those changes to your branch.

Reply:
great!